CRM/CiviAwards/Service/ApplicantManagementMenu.php

```diff
@@ -16,6 +16,8 @@ class CRM_CiviAwards_Service_ApplicantManagementMenu extends CRM_Civicase_Servic
    *   Menu ID.
    */
   protected function createCaseCategorySubmenus($caseTypeCategoryName, array $permissions, $caseCategoryMenuId) {
+    $labelForMenu = ucfirst($caseTypeCategoryName);
```

Comment:
Why not the same logic as the civicase repo? i.e strtolower combined with ucfirst, for consistency?

Reply:
Before it was not necessary because the parent method (the one on`CRM_Civicase_Service_CaseCategoryMenu`) was making lowercase the category name, but since now in civicase we are not doing that, the change is needed. Done: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/215/commits/7a46ba52b57bd3cd507aab5bcafeaec14351b53f